src/epub/text/colophon.xhtml

```diff
@@ -23,7 +23,7 @@
 			for<br/>
 			<a href="https://standardebooks.org/ebooks/p-g-wodehouse/golf-stories#transcriptions">Project Gutenberg</a><br/>
 			and on digital scans from the<br/>
-			<a href="https://standardebooks.org/ebooks/p-g-wodehouse/golf-stories#page-scans">Internet Archive</a>.</p>
```

Comment:
If there is more than one source, we say "various sources" https://standardebooks.org/manual/1.7.0/single-page#6.6.6.1.3.2.2

Reply:
Missed that in the manual. Time to re-read the whole thing...